packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -60,6 +60,11 @@ export interface StarkTableColumnProperties {
 	 */
 	label?: string;
 
+	/**
+	 * Value to be shown as the column's footer.
+	 */
+	footerValue?: any;
```

Comment:
Shouldn't it be a `string` here ?

Reply:
I don't know, do you think it'll alway be  a string. I think it can be a number too ?

Reply:
Mmm.. You're right! 😊 
Let's keep it like this for now

Reply:
oki ok